### ISSUE TITLE
Adduser et PHP8

### ISSUE THREAD

**Issue by Gpinpin** [phpbbmodders/phpbb-3.1-ext-adduser#23]
Hello
Do you plan to bring this extension in sync with PHP8?
This extension allowed me as an administrator to modulate new particular members.
Thank you already for bringing it to 3.1 ... But now we are in 3.3.3 ...
Gpinpin

translat google

**Comment by rmcgirr83** [comment]
Did you try it on PHP8?

**Comment by Gpinpin** [comment]
Yes and precisely it does not work in php8. Except phpbb version 3.3.3 is supposed to work with php 8 and 7.4. In 7.4 it still works ....

**Comment by rmcgirr83** [comment]
This should fix it for PHP 8

https://github.com/rmcgirr83/phpbb-3.1-ext-adduser/commit/db0611b4d4f4c68ae0cfc7b7327a2ab0a62b8b6b

**Comment by Gpinpin** [comment]
Thank you for your responsiveness I test in the evening
gp

**Comment by Gpinpin** [comment]
Good evening
local test no more bugs. I put this extension back in service.
Version noted 1.0.3 but we were at 1.0.4 so I think it should be noted 1.0.5 ;) :)
Thank you again for your responsiveness
GP
